tasksJS/task2.js

```diff
@@ -0,0 +1,133 @@
+//1 Welcome!
+//https://www.codewars.com/kata/577ff15ad648a14b780000e7
+//This task was in task1.js, I just changed a bit
+function greet(language) {
+  const GREETING = {
+    english: "Welcome",
+    czech: "Vitejte",
+    danish: "Velkomst",
+    dutch: "Welkom",
+    estonian: "Tere tulemast",
+    finnish: "Tervetuloa",
+    flemish: "Welgekomen",
+    french: "Bienvenue",
+    german: "Willkommen",
+    irish: "Failte",
+    italian: "Benvenuto",
+    latvian: "Gaidits",
+    lithuanian: "Laukiamas",
+    polish: "Witamy",
+    spanish: "Bienvenido",
+    swedish: "Valkommen",
+    welsh: "Croeso",
+
+    DEFAULT_GREETING: "Welcome",
+  };
+
+  return GREETING[language] || GREETING.DEFAULT_GREETING;
+}
+
+//2 Lario and Muigi Pipe Problem
+// https://www.codewars.com/kata/56b29582461215098d00000f
+//This task was in task1.js
+//Dima showed me best practice solution, i understood
+const pipeFix = (num) =>
+  Array(num[num.length - 1] - num[0] + 1)
+    .fill(num[0])
+    .map((element, index) => element + index);
+
+//3 Regular Ball Super Ball
+//https://www.codewars.com/kata/53f0f358b9cb376eca001079
+//This task was in task1.js, i propose second solution through "Class"
+class Ball {
+  constructor(ballType = "regular") {
+    this.ballType = ballType;
+  }
+}
+
+//4 Crash Override
+//https://www.codewars.com/kata/578c1e2edaa01a9a02000b7f
+//This task was in task1.js, I just changed a bit
+var firstName = { A: "Alpha", B: "Beta", C: "Cache" };
+var surname = { A: "Analogue", B: "Bomb", C: "Catalyst" };
+
+function aliasGen(firstname, lastname) {
+  const FIRST_NAME = firstname[0].toUpperCase();
+  const LAST_NAME = lastname[0].toUpperCase();
+  return !firstName[FIRST_NAME] || !surname[LAST_NAME]
+    ? "Your name must start with a letter from A - Z."
+    : firstName[FIRST_NAME] + " " + surname[LAST_NAME];
+}
```

Comment:
in this solution, you can use RegExp for practice. Because RegExp is a very powerful tool.
For Example:
if (/^\d/.test(first) || /^\d/.test(last)) { 
    return 'Your name must start with a letter from A - Z.';
  } else {
    return `${firstName[first[0].toUpperCase()]} ${surname[last[0].toUpperCase()]}`
  }